.travis.yml

```diff
@@ -45,13 +45,15 @@ jobs:
       script:
         - ddev dep verify
         - ddev manifest verify --include-extras
+        - ddev metadata verify
         - ddev test --cov
         - ddev test --bench || true
     - stage: test
       env: CHECK=datadog_checks_base PYTHON3=true
       script:
         - ddev dep verify
         - ddev manifest verify --include-extras
+        - ddev metadata verify
```

Comment:
Do you mean include extra so that it checks `integration-extras` as well or similar to manifest where optional fields are included? The former can be accomplished using the -e flag the way it's set up now eg. `ddev -e metadata verify`

Reply:
Ah yeah you are correct. I meant the former 👍 Sorry for the confusion. Good catch